www/css/cytube.css

```diff
@@ -266,16 +238,24 @@ li.ui-sortable-helper, li.ui-sortable-placeholder + li.queue_entry {
 
 .server-msg-disconnect {
     border: 1px solid #ff0000;
-    margin: 5px;
-    padding: 5px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    margin-top: 5px;
+    margin-right: 5px;
     color: #ff0000;
+    text-align: center;
```

Comment:
I just checked kick/mute messages, and the centering looks fine as long as the background colors are fixed.